src/Microsoft.AspNetCore.Razor.Language/CodeGeneration/HtmlAttributeWriter.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Razor.Language.Intermediate;
+
+namespace Microsoft.AspNetCore.Razor.Language.CodeGeneration
+{
+    internal abstract class HtmlAttributeWriter
```

Comment:
This is an internal class. Don't want to expose a way for people to override the HtmlAttribute code generation?